eigrpd/eigrp_packet.c

```diff
@@ -162,7 +162,7 @@ int eigrp_make_md5_digest(struct eigrp_interface *ei, struct stream *s,
 
 int eigrp_check_md5_digest(struct stream *s,
```

Comment:
looks like `s` is no longer being used in this function, so we can remove it?

Reply:
Sure，we can remove it